skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/ChatWindow.kt

```diff
@@ -114,6 +133,41 @@ fun ConversationField(modifier: Modifier = Modifier) {
   }
 }
 
+@Composable
+fun ChatBubble(message: Message, modifier: Modifier = Modifier) {
+  Box(
+    Modifier.fillMaxWidth()
```

Comment:
Does this need to fill the max width? It looks a little awkward in your video

Reply:
You have a point, I can adjust to make it less wide.

Reply:
Should I adjust it so that it's based on the input text? Here are some examples:
<img width="516" alt="Screenshot 2024-09-05 at 4 49 41 PM" src="https://github.com/user-attachments/assets/dc1a7f38-6538-4f3b-8abb-87bc67962246">

Reply:
yep that makes sense to me 👍

Reply:
though seems we should put the user's messages on one side and the bot's messages on the other?

Reply:
I put the updated video in the main description.